### ISSUE TITLE
Remove node's local min gas price 

### ISSUE THREAD

**Issue by ninabarbakadze** [celestiaorg/celestia-app#3896]
## Description

Currently, every validator node is able to set their local min gas price in app.toml. User's txs won't be accepted by the node if it doesn't meet that local threshold. On top of that we also have a global network min gas price that all transactions need to adhere to. 

Having two different thresholds to meet creates fragmented fee policies and makes it harder for the user to anticipate whether their transactions will be accepted or not by the network.

## Acceptance Criteria

- [ ] Get rid of node's local min gas price

**Comment by rootulp** [comment]
IIRC we had discussions about this change and we didn't get full support to remove the node's local min gas price. I think one reason to keep it was that validator's can locally configure their local min gas price faster than governance or a hard-fork can coordinate to change the network min gas price. 

So I think this issue has a prerequisite to get internal consensus that it should be removed.

**Comment by evan-forbes** [comment]
can we close this issue?

**Comment by ninabarbakadze** [comment]
yes it does not seem relevant after our most recent discussions related to min gas price
